### PR TITLE
Add Nexus Mods

### PR DESCRIPTION
**Pull request by DandelionSprout**
Apparently nexusmods.com supports 2FA by E-mail, as per https://help.nexusmods.com/article/74-two-factor-authentication-for-nexus-mods.

The site is ranked at a pretty impressive 494th on Alexa. I believe that the site deals with data (Criteria 2) in the form of uploaded mods, as well as personal image (Criteria 1) if the mod uploader has a large reputation. It also has a detailed documentation of its 2FA support.

### CODE REVIEW

File: _data/gaming.yml [hunk 1]
```diff
@@ -212,6 +211,13 @@ websites:
       twitter: MojangSupport
       img: mojang.png
       tfa: No
+      
+    - name: Nexus Mods
+      url: https://www.nexusmods.com/
+      img: nexusmods.png
+      tfa: Yes
+      email: Yes
+      doc: https://help.nexusmods.com/article/74-two-factor-authentication-for-nexus-mods
```
Comment (Carlgo11) [inline _data/gaming.yml]:
```suggestion
      doc: https://help.nexusmods.com/article/74--
```

**Review by Carlgo11 — DISMISSED** [review]

**Review by kmpoppe — CHANGES_REQUESTED** [review]
Looking at the logo I would suspect it could be shrunk a bit. Would you try running it through tinypng.com or any other service?

**Comment by DandelionSprout** [comment]
I have been using an OptiPNG plugin for Paint.NET, and I can't seem to get it below 1.6KB without heavily affecting the image quality (or 1.4KB in any way at all).

**Comment by DandelionSprout** [comment]
That being said, it appears that you (kmpoppe) were correct about the efficiency of TinyPNG after all.

**Comment by kmpoppe** [comment]
I have my bright moments, @dandelionsprout :-)
Thanks for checking tinypng again.

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**